### PR TITLE
 Translated new races (Sly, Replicon, Fulver), other minor updates.

### PR DESCRIPTION
**Pull request by Bestary**
Translated new races (Sly, Replicon, Fulver), edited other races, other minor updates.

@Cjkjvfnby, can you check again? :3 Sorry, there are looks like a lot of changes, but no need check them all scrupulously (mostly there was missed commas, orthographic errors, etc.), just check about new races.

### CODE REVIEW

**Comment by Bestary** [comment]
> This branch has conflicts that must be resolved

Hello again, dear problem. I missed you so much. :<

@Dilvish-fo, sorry, think I need your help again.
I tried to apply instructions you gave in my last PR, but can't remember this point (below), what kind of commit's number I need for cherry pick? (another incorrect merge from me, I think).

newcanvas2

**Comment by VPeruS** [comment]
You need to cherry-pick 5994103, a9cebf8, e6ebb11, fix merge conflict if any

**Comment by Bestary** [comment]
@Dilvish-fo, thank you again, seems that it's ok now. :) Actually there was needs just one last, other 2 makes conflicts (I resolved some).

**Comment by Dilvish-fo** [comment]
We are getting close here, but there really should only be one commit, not three-- those two "resolve conflicts" commits should not really be needed, and even if somehow those changes we needed in order for your "Translated new races (Sly..." commit to apply, that should all get rolled up into that one commit when you apply it (cherry pick it) and resolve conflicts as part of cherry picking.

Could you describe what process you followed that led to those two extra "Resolve conflicts" commits getting made first?

**Comment by Bestary** [comment]
> Could you describe what process you followed that led to those two extra "Resolve conflicts" commits getting made first?

There was actually just one conflict in one string. I made hard reset to master and cherry-pick first commit (5994103), then conflict appears. TortoiseGit proposed to resolve it by hands, I did it. Then I tried to cherry-pick next one (a9cebf8), and conflict (the same) appears again (and was again resolved by hands). Fird commit makes no conflict, but TortoiseGit asked to commit changes between cherry-picks. Actually those commits no need, they had no function and can be declined.

I learned more about cherry-pick function, now I understand how it works deeper. ^_^

**Comment by Dilvish-fo** [comment]
>  and cherry-pick first commit (5994103), then conflict appears. TortoiseGit proposed to resolve it by hands, I did it. Then I tried to cherry-pick next one (a9cebf8)

I suspected as much.. as I said above, the only commit you should be cherry picking here is the last one.  Could you please redo the hard reset to master and then ONLY cherry pick the last commit?  Thanks.

**Comment by Bestary** [comment]
> as I said above

Yes, I just not understanded correct this part:

> No, 5994103, and I think also a9cebf8, were part of the squash merge done in #2277,

thought you wants to ask me cherry-pick every of three :D Now there is just one right commit remained.

**Comment by geoffthemedio** [comment]
As @Cjkjvfnby hasn't commented, as the single commit now appears OK, will merge...

**Comment by geoffthemedio** [comment]
Because the first post requested you look at it:

> @Cjkjvfnby, can you check again?

**Comment by Bestary** [comment]
Sorry, guys, I still can't assign any person as a rewiever. >_<

**Comment by Dilvish-fo** [comment]
> Sorry, guys, I still can't assign any person as a rewiever. >_<

Ah -- I don't know if they've changed their rules, or changed their write-up on it, or if I just mis-read their page on assigning people before, but looking it over now I would no longer expect you to be able to.  I'll try to remember to assign your translation PRs to CJ when I see them submitted.

**Review by Cjkjvfnby — COMMENTED** [review]
I have finished post review.

File: default/stringtables/ru.txt [hunk 1]
```diff
@@ -6955,6 +6974,26 @@ SP_PHINNERT_DESC
 Их родная планета имеет низкую гравитацию и очень плотную атмосферу. Состав планеты уникален - в ней содержится очень низкий процент тяжёлых металлов, в то же время высок процент других тяжёлых элементов и благородных газов. Она также имеет стабильные тектонические образования, отчего кора у планеты очень плоская, и большая часть поверхности покрыта болотной, стоячей водой. Мир Финнертов - огромное, вечнозелёное болото.
 '''
 
+SP_REPLICON
+Репликон
+
+SP_REPLICON_GAMEPLAY_DESC
+'''Малообразованные самореплицирующие роботы.
+Prefer [[PT_RADIATED]] planets.
```
[the remark this answers is not in the document]
Comment (Bestary) [reply default/stringtables/ru.txt:6982]: Опаньки, пропустила ^^ Мерси. Добавлю со следующим pull-request-ом.

**Pull request merged**